.eslintrc.js

```diff
@@ -3,30 +3,21 @@ module.exports = {
     "node": true,
     "es6": true
   },
-  "plugins": ["lodash"],
-  "extends": ["eslint:recommended", "plugin:lodash/recommended"],
```

Comment:
I was wondering if you would want to keep the lodash plugin in this repository as it is used in this repository and the plugins will enforce us to use lodash functions.

Reply:
Thanks for the suggestion! We can keep the lodash plugin, but we had (and have) `"lodash/prefer-lodash-method": [0]` so it doesn't force us to use lodash functions.